src/Chapter1.hs

```diff
@@ -568,7 +573,14 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+isVowel :: Char -> Bool
+isVowel c
+    | c == 'a' = True
+    | c == 'e' = True
+    | c == 'i' = True
+    | c == 'o' = True
+    | c == 'u' = True -- a bit repetitive
```

Comment:
This is true! Don't worry, it is mostly to train usage of different constructions and syntax in Haskell 🙂 

There are a few more ways to write this function:

1. Just booleans:
    ```haskell
     isVowel c = c == 'a' || c == 'e' || ...
     ```
2. Using `elem` function:
    ```haskell
    isVowel c = elem c ['a', 'e', ...]
    ```
    
    or
    ```haskell
    isVowel c = elem c "ae..."
    ```
    
    One note in here, that sometimes, `elem` could be slower than the explicit pattern matching. I remember there were some benchmarks on one particular case, that showed how moving to pattern matching on each case separately drastically decrease time 🐎